azure/set_vs_toolchain.cmd

```diff
@@ -0,0 +1,71 @@
+@@echo on
```

Comment:
Can you add something like
```
REM Taken from conda-forge recipes licensed as follows
REM BSD-3-Clause
REM Copyright conda-forge contributors
```

Reply:
Good point - see: https://github.com/MacPython/numpy-wheels/pull/150